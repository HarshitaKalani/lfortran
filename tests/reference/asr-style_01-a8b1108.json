{
    "basename": "asr-style_01-a8b1108",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/warnings/style_01.f90",
    "infile_hash": "6d5ef2c8bd2e85b3d67e407d9df511c07558c442e27b4a5346240c23",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-style_01-a8b1108.stdout",
    "stdout_hash": "c15753a5c040b1461dc642d556650f11b02ef16ac75a869b3b28ed37",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}