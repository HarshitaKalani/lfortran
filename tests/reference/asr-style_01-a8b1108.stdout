(TranslationUnit
    (SymbolTable
        1
        {
            style_01:
                (Program
                    (SymbolTable
                        2
                        {
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    style_01
                    []
                    [(DoLoop
                        ()
                        ((Var 2 i)
                        (IntegerConstant 1 (Integer 4))
                        (IntegerConstant 10 (Integer 4))
                        ())
                        [(Print
                            [(Var 2 i)]
                            ()
                            ()
                        )]
                        []
                    )]
                )
        })
    []
)
